lib/streamlit/type_util.py

```diff
@@ -218,6 +218,7 @@ def get_fqn_type(obj: object) -> str:
 _SNOWPARK_DF_ROW_TYPE_STR: Final = "snowflake.snowpark.row.Row"
 _SNOWPARK_TABLE_TYPE_STR: Final = "snowflake.snowpark.table.Table"
 _PYSPARK_DF_TYPE_STR: Final = "pyspark.sql.dataframe.DataFrame"
+_MODIN_DF_TYPE_STR: Final = "modin.pandas.dataframe.DataFrame"
```

Comment:
```suggestion
_MODIN_DF_TYPE_STR: Final = "modin.pandas.DataFrame"
_MODIN_SERIES_TYPE_STR: Final = "modin.pandas.Series"
```

Reply:
Thanks, I also added support for Modin Series and Snowpark Pandas Dataframe & Series 👍  But I believe it needs to have the `.dataframe` part in the type name since that's the actual name if you do `str(type(modin_df))`.